templates/master/01-master-kubelet/_base/units/kubelet.yaml

```diff
@@ -10,6 +10,7 @@ contents: |
   Type=notify
   ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
   ExecStartPre=/bin/rm -f /var/lib/kubelet/cpu_manager_state
+  Environment="KUBELET_LOG_LEVEL=3"
```

Comment:
Did we test that we can override this in one of the environment files?

Reply:
yes. it works fine.